Doc/library/math.rst

```diff
@@ -82,6 +82,22 @@ Number-theoretic and representation functions
    should return an :class:`~numbers.Integral` value.
 
 
+.. function:: fma(x, y, z)
+
+   Fused multiply-add operation. Return ``(x * y) + z``, computed as though with
+   infinite precision and range followed by a single round to the ``float``
+   format. This operation often provides better accuracy than the direct
+   expression ``(x * y) + z``.
+
+   This function follows the specification of the fusedMultiplyAdd operation
+   described in the IEEE 754-2008 standard. The standard leaves one case
```

Comment:
I'd suggest updating this to say `IEEE 754` instead of `IEEE 754-2008` here. The 2008 standard is obsolete at this point, superseded by IEEE 754-2019. (We _could_ write `IEEE 754-2019`, but then we'd leave ourselves having to update _again_ in five years when the 2029 revision appears.)